controller/utils.go

```diff
@@ -13,6 +16,28 @@ import (
 	longhorn "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta2"
 )
 
+const (
+	podRecreateInitBackoff = 1 * time.Second
+	podRecreateMaxBackoff  = 120 * time.Second
+	backoffGCPeriod        = 12 * time.Hour
+)
+
+// newBackoff returns a flowcontrol.Backoff and starts a background GC loop.
```

Comment:
NIT: It might be worth mentioning the potential risk if this function is ever reused in a different context later. 
```suggestion
// newBackoff returns a flowcontrol.Backoff and starts a background GC loop.
//
// This function starts a background goroutine that cannot be gracefully stopped,
// making it suitable only for processes that run for the entire application's
// lifetime. Using it in short-lived or frequently called contexts will lead to
// goroutine leaks. For such cases, refactor to accept a context.Context for proper
// cleanup.
```